table/sst_file_writer.cc

```diff
@@ -36,12 +37,14 @@ struct SstFileWriter::Rep {
   ExternalSstFileInfo file_info;
   std::string column_family_name;
   InternalKey ikey;
+  ColumnFamilyHandle* cfh;
```

Comment:
Right now we never modify the data in the ColumnFamilyHandle, can this be a `const` pointer?